tensorflow/core/kernels/data/parallel_interleave_dataset_op.cc

```diff
@@ -604,10 +603,12 @@ class ParallelInterleaveDatasetOp : public UnaryDatasetOpKernel {
               continue;
             }
             DisableAutotune(ctx.get(), element->iterator.get());
+            ++num_calls_;
```

Comment:
`num_calls_` should not be used here because it determines the parallelism of CurrentElementsManager

Reply:
@jsimsa Could we add another variable (e.g. `future_num_calls_`) for FutureElementManager, so it will not affect the parallelism of CurrentElementsManager? If ok, we may also rename `num_calls_` to be `current_num_calls_`.

Reply:
That sounds reasonable.